File: Day 31/pitagoras.js
function point(x, y){
    this.x = x
    this.y = y
}

function distance(point1, point2){

    let d = Math.sqrt( Math.pow( (point1.x - point2.x) , 2) + Math.pow((point1.y - point2.y) , 2) )

    console.log(d)

}

const p1 = new point(2,3)
const p2 = new point(4,7)

distance(p1,p2)

File: Day 31/pitagoras.cpp
#include <iostream>
#include <math.h>

using namespace std;

class Point{
        
    public:
        
        int x;
        int y;

        Point(int X, int Y){
            x = X;
            y = Y;
        }

};

int main()
{
    Point p1(0,0);
    Point p2(3,4);

    int d = sqrt( pow( (p1.x - p2.x), 2 ) + pow( (p1.y - p2.y), 2 ) );
    cout << "The distance between the points is " << d << endl;

    return 0;
}
